packages/atlas/src/components/_inputs/WhiteListTextField/WhiteListTextField.tsx

```diff
@@ -28,6 +34,16 @@ export const WhiteListTextField: React.FC<WhiteListTextFieldProps> = ({ selected
   const [members, setMembers] = useState<Member[]>([])
   const client = useApolloClient()
   const [isError, setIsError] = useState(false)
+  const resolveAsset = useRawAssetResolver()
+
+  const getAvatarUri = (avatar?: BasicMembershipFieldsFragment['metadata']['avatar']) => {
+    if (avatar?.__typename === 'AvatarUri') {
+      return avatar.avatarUri
+    }
+    if (avatar?.__typename === 'AvatarObject' && avatar.avatarObject?.id) {
+      return resolveAsset(avatar.avatarObject?.id)?.url
```

Comment:
If you take a look at the raw asset resolver, you will notice that it will return an asset URL only if the asset has only been resolved, it doesn't schedule resolution for unknown assets. It'd be best to just pass either full member or full avatar object and use `useMemberAvatar`

Reply:
Should work now